axon-server-connector/src/test/java/org/axonframework/axonserver/connector/event/AxonServerUtils.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2010-2024. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.axonserver.connector.event;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Predicate;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AxonServerUtils {
+
+    public static void purgeEventsFromAxonServer(String hostname, int port) throws IOException {
+        final URL url = URI.create(String.format("http://%s:%d/v1/public/purge-events?targetContext=%s", hostname, port,
+                                                 URLEncoder.encode("default", StandardCharsets.UTF_8))).toURL();
+        HttpURLConnection connection = null;
+        try {
+            connection = (HttpURLConnection) url.openConnection();
+            connection.setDoOutput(true);
+            connection.setRequestMethod("DELETE");
+            connection.getInputStream().close();
+            assertFalse(connection.getResponseCode() >= 300);
+        } finally {
+            if (connection != null) {
+                connection.disconnect();
+            }
+        }
+        waitForContextsCondition(hostname, port, contexts -> contexts.contains("default"));
+    }
+
+    public static List<String> contexts(String hostname, int port) throws IOException {
```

Comment:
Scrap that last comment.
The contexts method is used to implement a check that the context exists after the purge. But since we call the purge API on Axon Server (previous versions deleted the context and recreated it), I wonder if all these methods are necessary.
On the other hand, if we decide to provide some APIs on the AxonServerTestContainer class, these could come in very handy again.